probe-rs/src/config/registry.rs

```diff
@@ -15,7 +15,7 @@ static REGISTRY: Lazy<Arc<Mutex<Registry>>> =
 #[derive(Debug, thiserror::Error)]
 pub enum RegistryError {
     /// The requested chip was not found in the registry.
-    #[error("The requested chip '{0}' was not found in the list of known targets.")]
+    #[error("The requested chip '{0}' was not found in the list of known targets. Run `probe-rs chip list` to see which chips are supported.")]
```

Comment:
This one is "problematic" because tools that are not `probe-rs` aka `cargo-flash` or `cargo-embed` or something homebrewn will not run the same command.

I am not sure what the best idea is. That's why we tried huge error inspection in `cargo-flash` back then which was not necessarily a nice to use solution.

What does @probe-rs/core think? :)

Reply:
Ah I see! That makes sense from a technical point of view. The reason I opened this PR is that I had some hickups (read: a typo 😅) during board bringup after a looong hiatus of not using `probe-rs`. I knew there was *some* way to check what a target name should look like & which targets are supported, but it took some poking around in the `probe-rs` source code to find what I was looking for (in part because I had only skimmed the `help` output). Even thought his is a tiny PR I do still think it would be helpful to offer some sort of relief to the "Ok, but what *do* you want then?" frustration that I ran into during the setup if at all technically possible.

Reply:
> What does @probe-rs/core think? :)

Will it help to add a link to the probe.rs docs, with different solutions for `probe-rs` vs. `cargo-xxxx` users?

Reply:
Links can also change 🤔